tests/get_coverage.yml

```diff
@@ -69,14 +71,17 @@
 - name: Gather coverage data
   hosts: all
   tasks:
-    - shell: "{{ coverage }} combine .coverage.*"
+    - name: gather coverage data
+      shell: "{{ coverage }} combine .coverage.*"
```

Comment:
Should we use `command` instead of `shell` here ?

Reply:
Because we are not using any symbol of `<, >, |, ;, &`  and command module is more secure( it will not be affected by the "user’s" environment).

Reply:
Not sure if `command` will work because of the wildcard match for `.coverage.*` but you could try it

Reply:
I failed with inventory `/usr/share/ansible/inventory/standard-inventory-qcow2`
```
[wenliang@localhost tests]$ ./get_coverage.sh /usr/share/ansible/inventory/standard-inventory-qcow2 ./tests_states_nm.yml             Getting coverage for ./tests_states_nm.yml on /usr/share/ansible/inventory/standard-inventory-qcow2

PLAY [Prepare for coverage extraction] **********************************************************

TASK [Gathering Facts] **************************************************************************
fatal: [/usr/share/ansible/inventory/standard-inventory-qcow2]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: Could not resolve hostname /usr/share/ansible/inventory/standard-inventory-qcow2: Name or service not known", "unreachable": true}

PLAY RECAP **************************************************************************************
/usr/share/ansible/inventory/standard-inventory-qcow2 : ok=0    changed=0    unreachable=1    failed=0    skipped=0    rescued=0    ignored=0 
```

Reply:
Can we test on qcow2 image or virtualBox image ?

Reply:
> Can we test on qcow2 image or virtualBox image ?

Testing with this tentative commit... https://github.com/linux-system-roles/network/pull/374/commits/5e5133a25dd831d95c6daa2255721e3974a57039

Reply:
I tested the wildcard match in both command and shell module:

Code:
```yaml
    - name: command grep
      command: "grep -R DOMA* /etc/sysconfig/network-scripts/"
      register: command_grep
  
    - name: shell grep
      shell: "grep -R DOMA* /etc/sysconfig/network-scripts/"
      register: shell_grep
```

Result (same result for command and shell module):
```
TASK [command grep] *********************************************************************************************************************************************************************************
task path: /home/wenliang/network/tests/playbooks/tests_eth_dns_support.yml:109
changed: [Fedora-Cloud-Base-33-20210106.0.x86_64.qcow2] => {"changed": true, "cmd": ["grep", "-R", "DOMA*", "/etc/sysconfig/network-scripts/"], "delta": "0:00:00.002010", "end": "2021-04-14 00:36:09.968410", "rc": 0, "start": "2021-04-14 00:36:09.966400", "stderr": "", "stderr_lines": [], "stdout": "/etc/sysconfig/network-scripts/ifcfg-ethtest0:DOMAIN=\"example.com example.org\"\n/etc/sysconfig/network-scripts/ifcfg-ethtest0:IPV6_DOMAIN=\"example.com example.org\"", "stdout_lines": ["/etc/sysconfig/network-scripts/ifcfg-ethtest0:DOMAIN=\"example.com example.org\"", "/etc/sysconfig/network-scripts/ifcfg-ethtest0:IPV6_DOMAIN=\"example.com example.org\""]}

TASK [shell grep] ***********************************************************************************************************************************************************************************
task path: /home/wenliang/network/tests/playbooks/tests_eth_dns_support.yml:113
changed: [Fedora-Cloud-Base-33-20210106.0.x86_64.qcow2] => {"changed": true, "cmd": "grep -R DOMA* /etc/sysconfig/network-scripts/", "delta": "0:00:00.002504", "end": "2021-04-14 00:36:10.312524", "rc": 0, "start": "2021-04-14 00:36:10.310020", "stderr": "", "stderr_lines": [], "stdout": "/etc/sysconfig/network-scripts/ifcfg-ethtest0:DOMAIN=\"example.com example.org\"\n/etc/sysconfig/network-scripts/ifcfg-ethtest0:IPV6_DOMAIN=\"example.com example.org\"", "stdout_lines": ["/etc/sysconfig/network-scripts/ifcfg-ethtest0:DOMAIN=\"example.com example.org\"", "/etc/sysconfig/network-scripts/ifcfg-ethtest0:IPV6_DOMAIN=\"example.com example.org\""]}
```

Reply:
Ok, then I'm squashing the commit. Thanks, @liangwen12year!

Reply:
> I tested the wildcard match in both command and shell module:

Great idea to test this!

> 
> Code:
> 
> ```yaml
>     - name: command grep
>       command: "grep -R DOMA* /etc/sysconfig/network-scripts/"
>       register: command_grep
>   
>     - name: shell grep
>       shell: "grep -R DOMA* /etc/sysconfig/network-scripts/"
>       register: shell_grep
> ```

`grep -R DOMA*` is not testing a wildcard but a regular expression. `*` means that the preceding item will be matched zero or more times (see man grep). To test pathname expansion, you would need to test something like `ls /etc/*`

Reply:
@nhosoi , I am afraid that  you need to change it back into `shell`

Code:
```yaml
    - name: shell ls
      shell: ls /etc/*
      register: shell_ls

    - name: command ls
      command: ls /etc/*
      register: command_ls
```      
Results(command module failed):
```
TASK [shell ls] *************************************************************************************************************************************************************************************
task path: /home/wenliang/network/tests/playbooks/tests_eth_dns_support.yml:109
changed: [Fedora-Cloud-Base-33-20210106.0.x86_64.qcow2] => {"changed": true, "cmd": "ls /etc/*", "delta": "0:00:00.013857", "end": "2021-04-14 12:55:46.510408", "rc": 0, "start": "2021-04-14 12:55:46.496551", "stderr": "", "stderr_lines": [], "stdout": "/etc/adjtime\n/etc/aliases\n/etc/bashrc\n/etc/bindresvport.blacklist\n/etc/chrony.conf\n/etc/chrony.keys\n/etc/crypttab\n/etc/csh.cshrc\n/etc/csh.login\n/etc/DIR_COLORS\n/etc/DIR_COLORS.lightbgcolor\n/etc/dracut.conf\n/etc/environment\n/etc/ethertypes\n/etc/exports\n/etc/extlinux.conf\n/etc/fedora-release\n/etc/filesystems\n/etc/fstab\n/etc/GREP_COLORS\n/etc/group\n/etc/group-\n/etc/grub2.cfg\n/etc/gshadow\n/etc/gshadow-\n/etc/host.conf\n/etc/hostname\n/etc/hosts\n/etc/idmapd.conf\n/etc/inittab\n/etc/inputrc\n/etc/issue\n/etc/issue.net\n/etc/krb5.conf\n/etc/ld.so.cache\n/etc/ld.so.conf\n/etc/libaudit.conf\n/etc/libuser.conf\n/etc/locale.conf\n/etc/localtime\n/etc/login.defs\n/etc/machine-id\n/etc/magic\n/etc/man_db.conf\n/etc/mke2fs.conf\n/etc/motd\n/etc/mtab\n/etc/mtools.conf\n/etc/netconfig\n/etc/networks\n/etc/nsswitch.conf\n/etc/nsswitch.conf.bak\n/etc/os-release\n/etc/passwd\n/etc/passwd-\n/etc/printcap\n/etc/profile\n/etc/protocols\n/etc/redhat-release\n/etc/resolv.conf\n/etc/rpc\n/etc/rsyncd.conf\n/etc/services\n/etc/sestatus.conf\n/etc/shadow\n/etc/shadow-\n/etc/shells\n/etc/subgid\n/etc/subgid-\n/etc/subuid\n/etc/subuid-\n/etc/sudo.conf\n/etc/sudoers\n/etc/sysctl.conf\n/etc/system-release\n/etc/system-release-cpe\n/etc/tcsd.conf\n/etc/vconsole.conf\n/etc/virc\n/etc/xattr.conf\n\n/etc/alternatives:\napropos\napropos.1.gz\ncifs-idmap-plugin\nifdown\nifup\nlibnssckbi.so.x86_64\nman\nman.1.gz\nsoelim\nsoelim.1.gz\nwhatis\nwhatis.1.gz\n\n/etc/audit:\nauditd.conf\naudit.rules\naudit-stop.rules\nplugins.d\nrules.d\n\n/etc/bash_completion.d:\n\n/etc/binfmt.d:\n\n/etc/cifs-utils:\nidmap-plugin\n\n/etc/cloud:\ncloud.cfg\ncloud.cfg.d\ntemplates\n\n/etc/console-login-helper-messages:\nissue.d\nmotd.d\n\n/etc/crypto-policies:\nback-ends\nconfig\nlocal.d\npolicies\nstate\n\n/etc/dbus-1:\nsession.conf\nsession.d\nsystem.conf\nsystem.d\n\n/etc/default:\ngrub\nuseradd\n\n/etc/depmod.d:\n\n/etc/dhcp:\ndhclient.d\ndhclient-exit-hooks.d\n\n/etc/dnf:\naliases.d\ndnf.conf\nmodules.d\nmodules.defaults.d\nplugins\nprotected.d\nvars\n\n/etc/dracut.conf.d:\n\n/etc/fonts:\nconf.d\n\n/etc/gcrypt:\nrandom.conf\n\n/etc/gnupg:\n\n/etc/groff:\nsite-font\nsite-tmac\n\n/etc/grub.d:\n00_header\n01_users\n08_fallback_counting\n10_linux\n10_reset_boot_success\n12_menu_auto_hide\n14_menu_show_once\n20_linux_xen\n20_ppc_terminfo\n30_os-prober\n30_uefi-firmware\n40_custom\n41_custom\nREADME\n\n/etc/gss:\nmech.d\n\n/etc/init.d:\nfunctions\nREADME\n\n/etc/iproute2:\nbpf_pinning\nematch_map\ngroup\nnl_protos\nrt_dsfield\nrt_protos\nrt_realms\nrt_scopes\nrt_tables\n\n/etc/issue.d:\n40_console-login-helper-messages.issue\n\n/etc/kernel:\ninstall.d\n\n/etc/krb5.conf.d:\ncrypto-policies\nkcm_default_ccache\n\n/etc/ld.so.conf.d:\n\n/etc/libibverbs.d:\nbnxt_re.driver\ncxgb4.driver\nefa.driver\nhfi1verbs.driver\nhns.driver\ni40iw.driver\nipathverbs.driver\nmlx4.driver\nmlx5.driver\nmthca.driver\nocrdma.driver\nqedr.driver\nrxe.driver\nsiw.driver\nvmw_pvrdma.driver\n\n/etc/libnl:\nclassid\npktloc\n\n/etc/libreport:\nevents\nevents.d\nplugins\nworkflows.d\n\n/etc/libssh:\nlibssh_client.config\nlibssh_server.config\n\n/etc/logrotate.d:\nchrony\ndnf\nsssd\n\n/etc/modprobe.d:\nmlx4.conf\ntruescale.conf\n\n/etc/modules-load.d:\n\n/etc/motd.d:\n\n/etc/NetworkManager:\nconf.d\ndispatcher.d\ndnsmasq.d\ndnsmasq-shared.d\nNetworkManager.conf\nsystem-connections\n\n/etc/openldap:\ncerts\nldap.conf\n\n/etc/opt:\n\n/etc/pam.d:\nconfig-util\nfingerprint-auth\nlogin\nother\npasswd\npassword-auth\npostlogin\nremote\nrunuser\nrunuser-l\nsmartcard-auth\nsshd\nsssd-shadowutils\nsu\nsudo\nsudo-i\nsu-l\nsystem-auth\nsystemd-user\nvlock\n\n/etc/pkcs11:\nmodules\n\n/etc/pki:\nca-trust\njava\nrpm-gpg\ntls\n\n/etc/pm:\nconfig.d\npower.d\nsleep.d\n\n/etc/popt.d:\n\n/etc/prelink.conf.d:\ngrub2.conf\n\n/etc/profile.d:\ncolorgrep.csh\ncolorgrep.sh\ncolorls.csh\ncolorls.sh\ncolorxzgrep.csh\ncolorxzgrep.sh\ncolorzgrep.csh\ncolorzgrep.sh\nconsole-login-helper-messages-profile.sh\ncsh.local\ngawk.csh\ngawk.sh\nlang.csh\nlang.sh\nless.csh\nless.sh\nsh.local\nvi.csh\nvi.sh\nwhich2.csh\nwhich2.sh\n\n/etc/rc0.d:\n\n/etc/rc1.d:\n\n/etc/rc2.d:\n\n/etc/rc3.d:\n\n/etc/rc4.d:\n\n/etc/rc5.d:\n\n/etc/rc6.d:\n\n/etc/rc.d:\ninit.d\nrc0.d\nrc1.d\nrc2.d\nrc3.d\nrc4.d\nrc5.d\nrc6.d\n\n/etc/rdma:\nmlx4.conf\nmodules\n\n/etc/rpm:\n\n/etc/rsyslog.d:\n21-cloudinit.conf\n\n/etc/rwtab.d:\nsssd\n\n/etc/sasl2:\n\n/etc/security:\naccess.conf\nchroot.conf\nconsole.apps\nconsole.handlers\nconsole.perms\nconsole.perms.d\nfaillock.conf\ngroup.conf\nlimits.conf\nlimits.d\nnamespace.conf\nnamespace.d\nnamespace.init\nopasswd\npam_env.conf\npwquality.conf\npwquality.conf.d\nsepermit.conf\ntime.conf\n\n/etc/selinux:\nconfig\nsemanage.conf\ntargeted\n\n/etc/skel:\n\n/etc/ssh:\nmoduli\nssh_config\nssh_config.d\nsshd_config\nsshd_config.d\nssh_host_dsa_key\nssh_host_dsa_key.pub\nssh_host_ecdsa_key\nssh_host_ecdsa_key.pub\nssh_host_ed25519_key\nssh_host_ed25519_key.pub\nssh_host_rsa_key\nssh_host_rsa_key.pub\n\n/etc/ssl:\ncerts\n\n/etc/sssd:\nconf.d\npki\n\n/etc/statetab.d:\n\n/etc/sudoers.d:\n90-cloud-init-users\n\n/etc/swid:\nswidtags.d\n\n/etc/sysconfig:\nanaconda\nchronyd\nconsole\nkernel\nman-db\nmodules\nnetwork\nnetwork-scripts\nselinux\nsshd\n\n/etc/sysctl.d:\n99-sysctl.conf\n\n/etc/systemd:\ncoredump.conf\nhomed.conf\njournald.conf\nlogind.conf\nnetwork\nnetworkd.conf\npstore.conf\nresolved.conf\nsleep.conf\nsystem\nsystem.conf\ntimesyncd.conf\nuser\nuser.conf\n\n/etc/terminfo:\n\n/etc/tmpfiles.d:\n\n/etc/tpm2-tss:\nfapi-config.json\nfapi-profiles\n\n/etc/udev:\nhwdb.bin\nhwdb.d\nrules.d\nudev.conf\n\n/etc/unbound:\nicannbundle.pem\nroot.key\n\n/etc/X11:\napplnk\nfontpath.d\nxinit\nxorg.conf.d\n\n/etc/xdg:\nautostart\nsystemd\n\n/etc/yum.repos.d:\nfedora-cisco-openh264.repo\nfedora-modular.repo\nfedora.repo\nfedora-updates-modular.repo\nfedora-updates.repo\nfedora-updates-testing-modular.repo\nfedora-updates-testing.repo", "stdout_lines": ["/etc/adjtime", "/etc/aliases", "/etc/bashrc", "/etc/bindresvport.blacklist", "/etc/chrony.conf", "/etc/chrony.keys", "/etc/crypttab", "/etc/csh.cshrc", "/etc/csh.login", "/etc/DIR_COLORS", "/etc/DIR_COLORS.lightbgcolor", "/etc/dracut.conf", "/etc/environment", "/etc/ethertypes", "/etc/exports", "/etc/extlinux.conf", "/etc/fedora-release", "/etc/filesystems", "/etc/fstab", "/etc/GREP_COLORS", "/etc/group", "/etc/group-", "/etc/grub2.cfg", "/etc/gshadow", "/etc/gshadow-", "/etc/host.conf", "/etc/hostname", "/etc/hosts", "/etc/idmapd.conf", "/etc/inittab", "/etc/inputrc", "/etc/issue", "/etc/issue.net", "/etc/krb5.conf", "/etc/ld.so.cache", "/etc/ld.so.conf", "/etc/libaudit.conf", "/etc/libuser.conf", "/etc/locale.conf", "/etc/localtime", "/etc/login.defs", "/etc/machine-id", "/etc/magic", "/etc/man_db.conf", "/etc/mke2fs.conf", "/etc/motd", "/etc/mtab", "/etc/mtools.conf", "/etc/netconfig", "/etc/networks", "/etc/nsswitch.conf", "/etc/nsswitch.conf.bak", "/etc/os-release", "/etc/passwd", "/etc/passwd-", "/etc/printcap", "/etc/profile", "/etc/protocols", "/etc/redhat-release", "/etc/resolv.conf", "/etc/rpc", "/etc/rsyncd.conf", "/etc/services", "/etc/sestatus.conf", "/etc/shadow", "/etc/shadow-", "/etc/shells", "/etc/subgid", "/etc/subgid-", "/etc/subuid", "/etc/subuid-", "/etc/sudo.conf", "/etc/sudoers", "/etc/sysctl.conf", "/etc/system-release", "/etc/system-release-cpe", "/etc/tcsd.conf", "/etc/vconsole.conf", "/etc/virc", "/etc/xattr.conf", "", "/etc/alternatives:", "apropos", "apropos.1.gz", "cifs-idmap-plugin", "ifdown", "ifup", "libnssckbi.so.x86_64", "man", "man.1.gz", "soelim", "soelim.1.gz", "whatis", "whatis.1.gz", "", "/etc/audit:", "auditd.conf", "audit.rules", "audit-stop.rules", "plugins.d", "rules.d", "", "/etc/bash_completion.d:", "", "/etc/binfmt.d:", "", "/etc/cifs-utils:", "idmap-plugin", "", "/etc/cloud:", "cloud.cfg", "cloud.cfg.d", "templates", "", "/etc/console-login-helper-messages:", "issue.d", "motd.d", "", "/etc/crypto-policies:", "back-ends", "config", "local.d", "policies", "state", "", "/etc/dbus-1:", "session.conf", "session.d", "system.conf", "system.d", "", "/etc/default:", "grub", "useradd", "", "/etc/depmod.d:", "", "/etc/dhcp:", "dhclient.d", "dhclient-exit-hooks.d", "", "/etc/dnf:", "aliases.d", "dnf.conf", "modules.d", "modules.defaults.d", "plugins", "protected.d", "vars", "", "/etc/dracut.conf.d:", "", "/etc/fonts:", "conf.d", "", "/etc/gcrypt:", "random.conf", "", "/etc/gnupg:", "", "/etc/groff:", "site-font", "site-tmac", "", "/etc/grub.d:", "00_header", "01_users", "08_fallback_counting", "10_linux", "10_reset_boot_success", "12_menu_auto_hide", "14_menu_show_once", "20_linux_xen", "20_ppc_terminfo", "30_os-prober", "30_uefi-firmware", "40_custom", "41_custom", "README", "", "/etc/gss:", "mech.d", "", "/etc/init.d:", "functions", "README", "", "/etc/iproute2:", "bpf_pinning", "ematch_map", "group", "nl_protos", "rt_dsfield", "rt_protos", "rt_realms", "rt_scopes", "rt_tables", "", "/etc/issue.d:", "40_console-login-helper-messages.issue", "", "/etc/kernel:", "install.d", "", "/etc/krb5.conf.d:", "crypto-policies", "kcm_default_ccache", "", "/etc/ld.so.conf.d:", "", "/etc/libibverbs.d:", "bnxt_re.driver", "cxgb4.driver", "efa.driver", "hfi1verbs.driver", "hns.driver", "i40iw.driver", "ipathverbs.driver", "mlx4.driver", "mlx5.driver", "mthca.driver", "ocrdma.driver", "qedr.driver", "rxe.driver", "siw.driver", "vmw_pvrdma.driver", "", "/etc/libnl:", "classid", "pktloc", "", "/etc/libreport:", "events", "events.d", "plugins", "workflows.d", "", "/etc/libssh:", "libssh_client.config", "libssh_server.config", "", "/etc/logrotate.d:", "chrony", "dnf", "sssd", "", "/etc/modprobe.d:", "mlx4.conf", "truescale.conf", "", "/etc/modules-load.d:", "", "/etc/motd.d:", "", "/etc/NetworkManager:", "conf.d", "dispatcher.d", "dnsmasq.d", "dnsmasq-shared.d", "NetworkManager.conf", "system-connections", "", "/etc/openldap:", "certs", "ldap.conf", "", "/etc/opt:", "", "/etc/pam.d:", "config-util", "fingerprint-auth", "login", "other", "passwd", "password-auth", "postlogin", "remote", "runuser", "runuser-l", "smartcard-auth", "sshd", "sssd-shadowutils", "su", "sudo", "sudo-i", "su-l", "system-auth", "systemd-user", "vlock", "", "/etc/pkcs11:", "modules", "", "/etc/pki:", "ca-trust", "java", "rpm-gpg", "tls", "", "/etc/pm:", "config.d", "power.d", "sleep.d", "", "/etc/popt.d:", "", "/etc/prelink.conf.d:", "grub2.conf", "", "/etc/profile.d:", "colorgrep.csh", "colorgrep.sh", "colorls.csh", "colorls.sh", "colorxzgrep.csh", "colorxzgrep.sh", "colorzgrep.csh", "colorzgrep.sh", "console-login-helper-messages-profile.sh", "csh.local", "gawk.csh", "gawk.sh", "lang.csh", "lang.sh", "less.csh", "less.sh", "sh.local", "vi.csh", "vi.sh", "which2.csh", "which2.sh", "", "/etc/rc0.d:", "", "/etc/rc1.d:", "", "/etc/rc2.d:", "", "/etc/rc3.d:", "", "/etc/rc4.d:", "", "/etc/rc5.d:", "", "/etc/rc6.d:", "", "/etc/rc.d:", "init.d", "rc0.d", "rc1.d", "rc2.d", "rc3.d", "rc4.d", "rc5.d", "rc6.d", "", "/etc/rdma:", "mlx4.conf", "modules", "", "/etc/rpm:", "", "/etc/rsyslog.d:", "21-cloudinit.conf", "", "/etc/rwtab.d:", "sssd", "", "/etc/sasl2:", "", "/etc/security:", "access.conf", "chroot.conf", "console.apps", "console.handlers", "console.perms", "console.perms.d", "faillock.conf", "group.conf", "limits.conf", "limits.d", "namespace.conf", "namespace.d", "namespace.init", "opasswd", "pam_env.conf", "pwquality.conf", "pwquality.conf.d", "sepermit.conf", "time.conf", "", "/etc/selinux:", "config", "semanage.conf", "targeted", "", "/etc/skel:", "", "/etc/ssh:", "moduli", "ssh_config", "ssh_config.d", "sshd_config", "sshd_config.d", "ssh_host_dsa_key", "ssh_host_dsa_key.pub", "ssh_host_ecdsa_key", "ssh_host_ecdsa_key.pub", "ssh_host_ed25519_key", "ssh_host_ed25519_key.pub", "ssh_host_rsa_key", "ssh_host_rsa_key.pub", "", "/etc/ssl:", "certs", "", "/etc/sssd:", "conf.d", "pki", "", "/etc/statetab.d:", "", "/etc/sudoers.d:", "90-cloud-init-users", "", "/etc/swid:", "swidtags.d", "", "/etc/sysconfig:", "anaconda", "chronyd", "console", "kernel", "man-db", "modules", "network", "network-scripts", "selinux", "sshd", "", "/etc/sysctl.d:", "99-sysctl.conf", "", "/etc/systemd:", "coredump.conf", "homed.conf", "journald.conf", "logind.conf", "network", "networkd.conf", "pstore.conf", "resolved.conf", "sleep.conf", "system", "system.conf", "timesyncd.conf", "user", "user.conf", "", "/etc/terminfo:", "", "/etc/tmpfiles.d:", "", "/etc/tpm2-tss:", "fapi-config.json", "fapi-profiles", "", "/etc/udev:", "hwdb.bin", "hwdb.d", "rules.d", "udev.conf", "", "/etc/unbound:", "icannbundle.pem", "root.key", "", "/etc/X11:", "applnk", "fontpath.d", "xinit", "xorg.conf.d", "", "/etc/xdg:", "autostart", "systemd", "", "/etc/yum.repos.d:", "fedora-cisco-openh264.repo", "fedora-modular.repo", "fedora.repo", "fedora-updates-modular.repo", "fedora-updates.repo", "fedora-updates-testing-modular.repo", "fedora-updates-testing.repo"]}

TASK [command ls] ***********************************************************************************************************************************************************************************
task path: /home/wenliang/network/tests/playbooks/tests_eth_dns_support.yml:113
fatal: [Fedora-Cloud-Base-33-20210106.0.x86_64.qcow2]: FAILED! => {"changed": true, "cmd": ["ls", "/etc/*"], "delta": "0:00:00.002096", "end": "2021-04-14 12:55:46.794685", "msg": "non-zero return code", "rc": 2, "start": "2021-04-14 12:55:46.792589", "stderr": "ls: cannot access '/etc/*': No such file or directory", "stderr_lines": ["ls: cannot access '/etc/*': No such file or directory"], "stdout": "", "stdout_lines": []}
```